qcodes/instrument_drivers/rohde_schwarz/ZNB20.py

```diff
@@ -1,20 +1,23 @@
+import time
+import logging
+import numpy as np
+
 from qcodes import VisaInstrument
 from qcodes.utils import validators as vals
 from cmath import phase
-import numpy as np
 from qcodes import Parameter
 
 
 class FrequencySweep(Parameter):
     """
-    Hardware controlled parameter class for Rohde Schwarz RSZNB20 trace.
+    Hardware controlled parameter class for Rohde Schwarz RSZNB20/8 trace.
 
     Instrument returns an list of transmission data in the form of a list of
     complex numbers taken from a frequency sweep.
 
-    TODO:
-      - ability to choose for abs or db in magnitude return
+    TODO(nataliejpg): ability to choose for abs or db in magnitude return
     """
+
```

Comment:
:unicorn:  missing class docstring, possibly obiovus but start stop npts are not super obvious from the outside